tests/integration/conftest.py

```diff
@@ -587,19 +854,39 @@ def add_test_case(self,
 
         if test_case.state == 'skipped':
             self._skip_count += 1
-        elif test_case.state == 'failed':
+        elif test_case.state == 'failure':
```

Comment:
Should the doc-string for the `failure_count` property be updated to reflect the new "failure" over "failed" value?

Reply:
@johnduarte Doc string updated in last commit.